libnmstate/schema.py

```diff
@@ -288,5 +288,20 @@ class OVSBridge(object):
 
     PORT_SUBTREE = 'port'
     PORT_NAME = 'name'
-    PORT_VLAN_MODE = 'vlan-mode'
-    PORT_ACCESS_TAG = 'access-tag'
+
+    class Port:
+        VLAN_SUBTREE = 'vlan'
+
+        class Vlan:
+            TRUNK_TAGS = 'trunk-tags'
+            TAG = 'tag'
+            ENABLE_NATIVE = 'enable-native'
+            TYPE = 'type'
```

Comment:
You set it to MODE in the other PR. Why the difference?

Reply:
Yea, I missed this and many other changes.
Fixed now.